install/kubernetes/cilium/templates/cilium-agent-daemonset.yaml

```diff
@@ -110,7 +128,11 @@ spec:
           # The initial delay for the liveness probe is intentionally large to
```

Comment:
this comment might cause some confusion for k8s 1.20 (i.e. initial delay is 0, but the comment is talking about `intentionally large`. Maybe move it inside the same block when value is set to 120.

Reply:
Indeed, this is fixed now.